optimizely/notification/entities.go

```diff
@@ -0,0 +1,34 @@
+/****************************************************************************
+ * Copyright 2019, Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+
+package notification
+
+import "github.com/optimizely/go-sdk/optimizely/entities"
+
+// Type is the type of notification
+type Type string
+
+const (
+	// Decision notification type
+	Decision Type = "decision"
+)
+
+// DecisionNotification is a notification triggered when a decision is made
```

Comment:
nit. I think we can do a better job documenting `DecisionNotification`

Reply:
Will do.